themes/ddbasic/sass/components/ding-carousel.scss

```diff
@@ -0,0 +1,224 @@
+/**
+ * Carousel
+ */
+
+@import '../base.scss';
+
+.carousel-tabs {
```

Comment:
In the spirit of the new frontend structure I think the class names for this should be updated with more structured naming and less nesting.

Reply:
You might be right but for now this is what I could do. So it most be sent on to the clean up theme project.